.changeset/tasty-forks-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs-node': patch
+---
+
+Updated getRepoUrlFromLocationAnnotation to check for harness scm integration
```

Comment:
```suggestion
Updated `getRepoUrlFromLocationAnnotation` to check for Harness SCM integration
```

Reply:
thanks for review, fixed changeset @awanlin